main.py

```diff
@@ -13,9 +14,34 @@
                                              initial_distribution=v,
                                              num_stages=N, stopping_time=tau).create()
 
-tree.bulls_eye_plot(dot_size=5, radius=300)
+# tree.bulls_eye_plot(dot_size=5, radius=300)
+#
+# tree.set_data_at_node(4, {"a": 1})
+# print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+#
+# print(tree)
 
-tree.set_data_at_node(4, {"a": 1})
-print(sum(tree.probability_of_node(tree.nodes_at_stage(8))))
+# RAOCP generation -----------------------------------------------------------------------------------------------------
 
-print(tree)
+Aw1 = np.eye(2)
+Aw2 = 2*np.eye(2)
+Aw3 = 3*np.eye(2)
+As = [Aw1, Aw2, Aw3]  # n x n matrices
+
+Bw1 = np.eye(2)
+Bw2 = 2*np.eye(2)
+Bw3 = 3*np.eye(2)
+Bs = [Bw1, Bw2, Bw3]  # n x u matrices
+
+cost_type = "Quadratic"
```

Comment:
Don't use strings to describe the type of an object. In OOP all objects have a type. Moreover, the user (and we) will be confused: is it `"Quadratic"`, `"quadratic"`, `"quad"`? And what other options are there? In some cases, it is useful to use enums (not here I think).